client/my-sites/pages/page-list.jsx

```diff
@@ -375,10 +332,6 @@ class Pages extends Component {
 	renderChronological( { pages, site, showPublishedStatus } ) {
 		const { search, status } = this.props.query;
 
-		if ( ! search ) {
-			// we're listing in reverse chrono. use the markers.
-			pages = this._insertTimeMarkers( pages );
```

Comment:
What's the reasoning behind this removal? It prevents the pages from being grouped by day.

Before | After
--- | ---
<img width="969" alt="Screen Shot 2020-04-06 at 12 57 34" src="https://user-images.githubusercontent.com/1233880/78551876-bdf04b00-7806-11ea-917f-8abc5e4c7c28.png"> | <img width="962" alt="Screen Shot 2020-04-06 at 12 57 41" src="https://user-images.githubusercontent.com/1233880/78551933-d95b5600-7806-11ea-8756-4c87b106cfe3.png">

Reply:
It is linked to @gwwar comment and @lcollette answer [here](https://github.com/Automattic/wp-calypso/issues/40553#issuecomment-608254297). Have I misunderstood something? 🤔

Reply:
Thanks! I was missing that context. Changes look good then 👍